artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -2937,6 +2937,13 @@
                </xsd:documentation>
             </xsd:annotation>
          </xsd:element>
+         <xsd:element name="data-source-time-margin" type="xsd:string" minOccurs="0" maxOccurs="1">
```

Comment:
Should probably be a integer or long not a string, also should have default defined in schema